### DISCUSSION TITLE
Mistake on competition page

### DISCUSSION THREAD

**Discussion by Abinashbunty** [esa/SpOC2#2] [General]
Hi @esa members. 😄 I was just going through the competition page and saw that there is a mistake in the date. In the `Scope of the Competition` section on [this page](https://www.esa.int/gsp/ACT/projects/spoc-2023), the date is written as _1 April 2022_. Instead, it should be _**1 April 2023**_. 

I didn't know whom to inform, but thought of putting it in Discussions section of the repo. 😅

**Comment by EmBlazquez** [comment]
Thanks @Abinashbunty , fixed :)
